engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/views/apply-errata-confirm.html

```diff
@@ -0,0 +1,64 @@
+<span page-title ng-model="errata">{{ 'Apply Errata' | translate }}</span>
+
+<section class="col-md-4" ng-hide="selectedContentHosts">
+  <p class="alert alert-warning" translate>You must select at least one Content Host in order to apply Errata.</p>
+</section>
+
+<section class="details details-full" ng-show="selectedContentHosts">
+  <div bst-alert success-messages="successMessages" error-messages="errorMessages"></div>
+  <h3 ng-show="errata" translate>Apply {{ errata.errata_id }}</h3>
+
+  <section ng-hide="updates">
+    <p translate ng-show="errata">
+      Apply {{ errata.errata_id }} to {{ selectedContentHosts.included.ids.length }} Content Host(s)?
+    </p>
+    <p translate ng-hide="errata">
+      Apply {{ errataIds.length }} Errata to {{ selectedContentHosts.included.ids.length }} Content Host(s)?
+    </p>
+  </section>
+
+  <section ng-show="updates">
+    <p translate>
+      These Errata were not available in your published Content View Versions running on the selected hosts.  The new Content View Versions (specified below)
+      will be created which will make this Errata available in the host's Environment.  This new version will replace the current version in your host's Lifecycle
+      Environment.  To install these errata immediately on hosts after publishing check the box below.
```

Comment:
Ah, that's leftover from when I had the checkbox you commented on last time still there.  Will update.